### PR TITLE
Rename shake & vs30

### PR DESCRIPTION
**Pull request by joa-quim**
Rename these supplemental modules.

### CODE REVIEW

File: doc/rst/source/reference/supplemental-packages.rst [hunk 1]
```diff
@@ -89,8 +89,8 @@ This package contains the programs
 :doc:`meca </supplements/seis/meca>`,
 :doc:`polar </supplements/seis/polar>`,
 :doc:`sac </supplements/seis/sac>`,
-:doc:`sac </supplements/seis/vs30>`, and
-:doc:`sac </supplements/seis/shake>`, which are used by seismologists
+:doc:`sac </supplements/seis/grdvs30>`, and
+:doc:`sac </supplements/seis/grdshake>`, which are used by seismologists
```
Comment (seisman) [inline doc/rst/source/reference/supplemental-packages.rst:93]:
```suggestion
:doc:`grdvs30 </supplements/seis/grdvs30>`, and
:doc:`grdshake </supplements/seis/grdshake>`, which are used by seismologists
```

**Review by PaulWessel — APPROVED** [review]
Pretty good, but you need to do two things:

1. chase the shake_purpose and vs30_purpose to their new names.
2. Remember to run the src/gmt_make_module_purpose.sh to update those strings. [we should look into having CMake figure out that his needs to be done since I tend to forget.]

**Comment by PaulWessel** [comment]
Oh, note your man pages repeat the option, ell |-L|-L ... same for some others.

**Comment by joa-quim** [comment]
It now says it has conflicts on the no longer existing files (``shake.c, vs30.c``)??? What conflicts?

**Comment by PaulWessel** [comment]
I jus merged master into the branch and looks now?

**Pull request merged**